contrib/completion/bash/docker

```diff
@@ -347,6 +352,16 @@ __docker_complete_log_driver_options() {
 			" -- "${cur#=}" ) )
 			return
 			;;
+		*splunk-url=*)
+			COMPREPLY=( $( compgen -W "http https" -S "://" -- "${cur#=}" ) )
```

Comment:
This does not behave as expected because `:` is special (contained in `COMP_WORDBREAKS`).
`docker run --log-opt splunk-url=http:` will not complete anything.

Please change to

    COMPREPLY=( $( compgen -W "http:// https://" -- "${cur#=}" ) )
    compopt -o nospace
    __ltrim_colon_completions "${cur}"
    return
    ;;